packages/compass-saved-aggregations-queries/.eslintrc.js

```diff
@@ -6,4 +6,12 @@ module.exports = {
     tsconfigRootDir: __dirname,
     project: ['./tsconfig-lint.json'],
   },
+  overrides: [
+    {
+      files: ['**/*.ts'],
+      rules: {
+        '@typescript-eslint/switch-exhaustiveness-check': 'error',
```

Comment:
We could also enable this everywhere, but I imagine that could cause unintended failures we have to fix and I'd rather do that separately.

Reply:
Can you file a ticket to do this if we're not trying to do this right away?

Reply:
https://jira.mongodb.org/browse/COMPASS-9165